### ISSUE TITLE
Problem in BuildLogs

### ISSUE THREAD

**Issue by P-yiush07** [adrianhajdin/ecommerce#7]
When i am pushing the code to gitHub, after starting of build process, i am getting this kind of errors

**<w> [webpack.cache.PackFileCacheStrategy] Caching failed for pack: Error: No serializer registered for ConcatSource**

and mostly this kind of errors:

**23:1 Error: Replace `⏎import·classes·from·'./index.module.scss';` with `import·classes·from·'./index.module.scss'` prettier/prettier
--
 
[2023-12-09T14:19:25] | │ 24:1 Error: Run autofix to sort these imports! simple-import-sort/imports
 
[2023-12-09T14:19:25] | │ 26:52 Error: Delete `⏎` prettier/prettier
 
[2023-12-09T14:19:25] | │ 66:9 Error: Delete `⏎··········` prettier/prettier
 
[2023-12-09T14:19:25] | │ 74:1 Error: Delete `··` prettier/prettier
 
[2023-12-09T14:19:25] | │ 120:2 Error: Insert `⏎` prettier/prettier
 
[2023-12-09T14:19:25] | │ 120:2 Error: Newline required at end of file but not found. eol-last**

in the end i am getting this

error Command failed with exit code 1

and then build is getting failed

**Comment by jurabek105** [comment]
Having the same problem, didnt you find any solution?

**Comment by P-yiush07** [comment]
> Having the same problem, didnt you find any solution?

Yeah i found, install es lint extension in your vs code, and fix all of the error one by one, by auto fixing. And then push it to github and then start the build, it will get resolved

**Comment by jurabek105** [comment]
> > Having the same problem, didnt you find any solution?
> 
> Yeah i found, install es lint extension in your vs code, and fix all of the error one by one, by auto fixing. And then push it to github and then start the build, it will get resolved

thanks a lot
